src/Session.ts

```diff
@@ -100,10 +148,20 @@ export class Session extends EventEmitter {
     return sessionInfo;
   };
 
+  /**
+   * Register a callback function to be called when a user completes login.
+   *
```

Comment:
```suggestion
   * Register a callback function to be called when a user completes login.
   *
   * The callback is called when {@link handleIncomingRedirect} completes successfully.
   *
```

Assuming that this is true? If not, a comment on when it _is_ called would be useful :)

Reply:
It is currently not, but it should be: that's a bug I'm willing to fix ^^